Source/Core/DolphinWX/ConfigMain.cpp

```diff
@@ -467,13 +468,20 @@ void CConfigMain::InitializeGUITooltips()
 
 	InterfaceLang->SetToolTip(_("Change the language of the user interface.\nRequires restart."));
 
-	// Audio tooltips
+	// Audio - Backend
 	BackendSelection->SetToolTip(_("Changing this will have no effect while the emulator is running!"));
 
+	// GameCube - IPL
+	GCSystemLang->SetToolTip(_("Sets the GameCube system language."));
+
 	// GameCube - Devices
-	GCEXIDevice[2]->SetToolTip(_("Serial Port 1 - This is the port which devices such as the net adapter use"));
+	GCEXIDevice[2]->SetToolTip(_("Serial Port 1 - This is the port which devices such as the net adapter use."));
+
+	// Wii - Misc
+	WiiSystemLang->SetToolTip(_("Sets the Wii system language."));
 
 	// Wii - Devices
+	WiiSDCard->SetToolTip(_("Saved to /Wii/SD.raw (Default size is 128mb)"));
```

Comment:
I thought it needed to be capitalized because it was in parentheses. My mistake. Will fix.